Tasks/MavenPreview/task.json

```diff
@@ -93,41 +84,24 @@
             "label": "CodeCoverage Tool",
             "required": false,
             "groupName": "codeCoverage",
-            "defaultValue": "JaCoCo",
-            "helpMarkDown": "Select the tool with which summary files have been generated",
+            "defaultValue": "NoCoverage",
+            "helpMarkDown": "Select the tool for evaluating code coverage. Only Jacoco support available currently.",
```

Comment:
Remove Only Jacoco support available currently. Gopi suggested same for ant.
And make it Select code coverage tool.

If there is some requirement for maven like  " jacocoant.jar " for ant. Add that suggestion to have that as prereq